lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -287,19 +287,18 @@ function dosomething_reportback_files_form_submit($form, &$form_state) {
     // Check to see if Rogue feature flag is enabled.
     if (variable_get('rogue_collection', FALSE)) {
       $rogue_item_id = dosomething_rogue_get_by_file_id($fid);
-
       // If there is a rogue_item_id in the dosomething_rogue_reportbacks table, send update to Rogue.
-      if ($rogue_item_id[0]->rogue_item_id) {
+      if ($rogue_item_id[0]->rogue_event_id) {
         if ($values['status'] === 'promoted' || $values['status'] === 'approved' || $values['status'] === 'excluded') {
           $rogue_status = 'accepted';
         } else {
           $rogue_status = 'rejected';
         }
 
         $data = [
-          'rogue_reportback_item_id' => $rogue_item_id[0]->rogue_item_id,
+          'rogue_event_id' => $rogue_item_id[0]->rogue_event_id,
           'status' => $rogue_status,
-          'reviewer' => dosomething_user_get_field('field_northstar_id'),
```

Comment:
@chloealee @katiecrane we do have a `reviews` table in rogue that was supposed to store stuff like this but I don't know if we have fully implemented that yet. I think we can talk more about how we want to handle this because we haven't really thought through it all yet.